docs/docs/Support/release-notes.mdx

```diff
@@ -50,6 +50,11 @@ For all changes, see the [Changelog](https://github.com/langflow-ai/langflow/rel
 
 ### New features and enhancements
 
+- Manage dependencies in the UI
```

Comment:
This feature will not be included in 1.6, but the Docling dependency is included with the Langflow OSS package.